src/lib/pages/workflow-stack-trace.svelte

```diff
@@ -81,7 +81,7 @@
       {#if workflow.isRunning && workers?.pollers?.length === 0}
         <p>
           To enable <Link
-            href="https://docs.temporal.io/concepts/what-is-a-query/#stack-trace-query"
+            href="https://docs.temporal.io/workflows#query"
```

Comment:
There is actually a `#stack-trace-query` anchor
```suggestion
            href="https://docs.temporal.io/workflows#stack-trace-query"
```